src/python/pants/engine/target_test.py

```diff
@@ -1043,46 +1055,83 @@ class IrrelevantSources(Sources):
 # -----------------------------------------------------------------------------------------------
 
 
-class FortranDependencies(Dependencies):
+class SmalltalkDependencies(Dependencies):
```

Comment:
I renamed from `Fortran` to `Smalltalk` because we need to introduce `Sources` subclass, and `FortranSources` was already claimed but implemented in a way we don't want.